### PR TITLE
a small fix in the delete all pods

### PR DESCRIPTION
**Pull request by ahmadbanijamali**
There was a typo in kubectl -n my-ns delete pod,svc --all

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
> 
> For pull requests on Chinese localization, set Base Branch to release-1.15
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by ahmadbanijamali** [comment]
The CLA is done!

**Comment by zacharysarah** [comment]
@ahmadbanijamali It looks like you still need to sign the CLA, or make sure you're logged in with the email you used to sign it.

**Comment by ahmadbanijamali** [comment]
@zacharysarah It is done now. Thanks!

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -272,7 +272,7 @@ kubectl scale --replicas=5 rc/foo rc/bar rc/baz                   # Scale multip
 kubectl delete -f ./pod.json                                              # Delete a pod using the type and name specified in pod.json
 kubectl delete pod,service baz foo                                        # Delete pods and services with same names "baz" and "foo"
 kubectl delete pods,services -l name=myLabel                              # Delete pods and services with label name=myLabel
-kubectl -n my-ns delete po,svc --all                                      # Delete all pods and services in namespace my-ns,
```
Comment (stewart-yu) [inline content/en/docs/reference/kubectl/cheatsheet.md]: `po` equal to `pod`

**Pull request merged**